app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -466,970 +466,776 @@ class ExplorationActivityTest {
+          .inRoot(isDialog())
+          .check(matches(isDisplayed()))
+      }
+      explorationDataController.stopPlayingExploration()
     }
-    explorationDataController.stopPlayingExploration()
-  }
 
-  @Test
-  fun testAudioWithNoConnection_openRatioExploration_clickAudioIcon_checkOpensNoConnectionDialog() {
-    setUpAudio()
-    launch<ExplorationActivity>(
-      createExplorationActivityIntent(
-        internalProfileId,
-        RATIOS_TOPIC_ID,
-        RATIOS_STORY_ID_0,
-        RATIOS_EXPLORATION_ID_0,
-        shouldSavePartialProgress = false
-      )
-    ).use {
-      explorationDataController.startPlayingExploration(
-        internalProfileId,
-        RATIOS_TOPIC_ID,
-        RATIOS_STORY_ID_0,
-        RATIOS_EXPLORATION_ID_0,
-        shouldSavePartialProgress = false,
-        explorationCheckpoint = ExplorationCheckpoint.getDefaultInstance()
-      )
-      networkConnectionUtil.setCurrentConnectionStatus(ProdConnectionStatus.NONE)
-      testCoroutineDispatchers.runCurrent()
-      onView(withId(R.id.action_audio_player)).perform(click())
-      onView(withText(context.getString(R.string.audio_dialog_offline_message)))
-        .inRoot(isDialog())
-        .check(matches(isDisplayed()))
+    @Test
+    fun testAudioWithCellular_openRatioExploration_clickAudioIcon_checkOpensCellularAudioDialog() {
+      setUpAudio()
+      launch<ExplorationActivity>(
+        createExplorationActivityIntent(
+          internalProfileId, RATIOS_TOPIC_ID,
+          RATIOS_STORY_ID_0, RATIOS_EXPLORATION_ID_0,
+          shouldSavePartialProgress = false
+        )
+      ).use {
+        explorationDataController.startPlayingExploration(
+          internalProfileId,
+          RATIOS_TOPIC_ID,
+          RATIOS_STORY_ID_0,
+          RATIOS_EXPLORATION_ID_0,
+          shouldSavePartialProgress = false,
+          explorationCheckpoint = ExplorationCheckpoint.getDefaultInstance()
+        )
+        networkConnectionUtil.setCurrentConnectionStatus(ProdConnectionStatus.CELLULAR)
+        testCoroutineDispatchers.runCurrent()
+        onView(withId(R.id.action_audio_player)).perform(click())
+        onView(withText(context.getString(R.string.cellular_data_alert_dialog_title)))
+          .inRoot(isDialog())
+          .check(matches(isDisplayed()))
+      }
+      explorationDataController.stopPlayingExploration()
     }
-    explorationDataController.stopPlayingExploration()
-  }
 
-  @Test
-  fun testAudioWithCellular_openRatioExploration_clickAudioIcon_checkOpensCellularAudioDialog() {
-    setUpAudio()
-    launch<ExplorationActivity>(
-      createExplorationActivityIntent(
-        internalProfileId, RATIOS_TOPIC_ID,
-        RATIOS_STORY_ID_0, RATIOS_EXPLORATION_ID_0,
-        shouldSavePartialProgress = false
-      )
-    ).use {
-      explorationDataController.startPlayingExploration(
+    @Test
+    fun testAudioCellular_ratioExp_audioIcon_configChange_opensCellularAudioDialog() {
```

Comment:
Here & elsewhere: what changed in all of these tests? I'm having a difficult time interpreting the deltas.

Reply:
@BenHenning I have just added  tests , 1st for checking of Opening help activity, 2nd for Options activity and 3rd for closing of bottomsheet, rest in the audio tests I have not made any changes I am myself confused why git is showing that as a change in my commits, even though the content is exact same.